tiles/src/main/java/com/protomaps/basemap/layers/Pois.java

```diff
@@ -355,7 +356,9 @@ public void processFeature(SourceFeature sf, FeatureCollector features) {
             }
           }
         } else if (kind.equals("cemetery") ||
-          kind.equals("school")) {
+          kind.equals("school") ||
+          kind.equals("allotments") ||
+          kind.equals("playground")) {
```

Comment:
Playground POIs should only appear at later zooms, like zoom 16.

(see also how Tilezen sets the value: https://github.com/tilezen/vector-datasource/blob/master/yaml/pois.yaml#L1315-L1324)

Reply:
Okay, I'll add a separate branch showing them from zoom level 16 onwards.